transactions/pendingtxtracker.go

```diff
@@ -139,6 +151,18 @@ func (tm *PendingTxTracker) fetchAndUpdateDB(ctx context.Context) bool {
 	return res
 }
 
+type nullableReceipt struct {
+	*types.Receipt
+}
+
+func (nr *nullableReceipt) UnmarshalJSON(data []byte) error {
+	if string(data) == "null" {
```

Comment:
not sure of the best practices etc in this codebase/Go in general but I generally prefer to use named variables and let the code speak then commenting etc. consider something like:
```
const transactionNotAvailable = string(data) == "null"
if  transactionNotAvailable {
	return nil
}
```

Reply:
Good idea, thanks for the heads up.